components/src/dynamo/trtllm/llm_engine.py

```
@@ -512,21 +513,46 @@ def _kv_events_poll_loop(self) -> None:
-                            "Failed to dispatch KV event; suppressing further "
-                            "tracebacks (last error: %s)",
-                            e,
-                        )
+            self._dispatch_kv_events(events)
+
+    def _dispatch_kv_events(self, events: list[dict[str, Any]]) -> None:
+        events_by_rank: dict[int, list[KvStoredEventInput | KvRemovedEventInput]] = {}
+        for event in events:
+            try:
+                normalized = self._normalize_kv_event(event)
+            except Exception as error:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)